### PR TITLE
Slim docker image

### PR DESCRIPTION
**Pull request by david-baylibre**
Base the dockerfile on slim python image.
This roughly reduces image size by 75%.

### CODE REVIEW

File: repo_resource/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 gitrepo==2.31.1
 ssh-agent-setup==2.0.1
+GitPython==3.1.31
```
Comment (makohoek) [inline repo_resource/requirements.txt:3]: This seems like an additional change that has nothing to do with changing the base docker image.
Could we clarify in the commit message why the additional requirement has been added?

**Review by makohoek — CHANGES_REQUESTED** [review]

File: repo_resource/requirements.txt (hunk 1 above)
Comment (david-baylibre) [reply repo_resource/requirements.txt]: Leftover from other branch, removed

Comment (makohoek) [reply repo_resource/requirements.txt]: Ok

**Review by makohoek — APPROVED** [review]
Looks good. I've ran the unit tests and they all passed, besides a minor problem with recent python versions.

**Pull request merged**